tests/phpunit/Metadata/Builder/ColumnBuilderTest.php

```diff
@@ -241,4 +241,17 @@ public function testForeignKeyAlreadySet(): void
         $this->expectExceptionMessage('Foreign key is already set.');
         $builder->addForeignKey();
     }
+
+    public function testTrimDisabled(): void
+    {
+        $name = ' '; // in MsSQL is one space valid column name
+
+        $builder = $this->createBuilder();
+        $builder->setName($name, false);
+        $builder->setType('INT');
+
+        $valueObject = $builder->build();
+        Assert::assertSame(' ', $valueObject->getName());
+        Assert::assertSame('empty_name', $valueObject->getSanitizedName());
```

Comment:
Nakoniec som sa rozhodol pre univerzalne riesenie, ... toto by malo fungovat za kazdych okolnosti, aj pri `SELECT *` a generovaní manifestu, ... aj pri `getTables` ... proste je to validny nazov stlpca, nemozeme ho ignorovat (lebo vznikne vela problemov).